frontend/src/metabase/visualizations/visualizations/CartesianChart/events.ts

```diff
@@ -174,9 +183,56 @@ export const getEventColumnsData = (
       };
     })
     .filter(isNotNull);
+
+  return dataPoints;
 };
 
-export const getStackedTooltipModel = (
+const getTooltipFooterData = (
+  chartModel: BaseCartesianChartModel,
+  display: string,
+  seriesIndex: number,
+  dataIndex: number,
+): DataPoint[] => {
+  if (display === "scatter" || !isTimeSeriesAxis(chartModel.xAxisModel)) {
+    return [];
+  }
+
+  const datum = chartModel.dataset[dataIndex];
+  const seriesModel = chartModel.seriesModels[seriesIndex];
+
+  const currentValue = datum[seriesModel.dataKey];
+  const currentDate = dayjs(String(datum[X_AXIS_DATA_KEY]));
```

Comment:
We have some custom parsing logic for timeseries row values, so it is better to use `parseTimestamp` from `frontend/src/metabase/lib/time-dayjs.ts` here and below